index.html

```diff
@@ -0,0 +1,108 @@
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="last-list-child"><a href="#">Concert Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="headline-title">
+        <p>"ITI-CCR KUNST Presents!"</p>
+        <h1>PRAISYDRAM 2022</h1>
+      </div>
+      <p class="headline-details">A wonderful opportunity to witness the greatness of God in our lives as his most inportant creation.
+        Fun packed activities to cherish for the rest of your live. You just can't miss this!
+      </p>
+      <div class="headline-bottom">
+        <p class="bottom-top">
+          2022.03.18(FRI) ~ 19(SAT)
+        </p>
+        <p>
+          @ KNUST Parade Grounds
+        </p>    
+      </div>    
+    </section>
+    <section class="main-program">
+      <h2 class="program-title">Main Programs</h2>
+      <span class="border-span"></span>
+      <div class="program-cards">
+        <article class="card">
+          <i class="fas fa-microphone-alt fa-2x"></i>
+          <h3>Praises & Worship</h3>
+          <p>Praise and worship songs by invited singing groups</p>
+        </article>
+        <article class="card">
+          <i class="fas fa-theater-masks fa-2x"></i>
+          <h3>Drama</h3>
+          <p>A well reheased play by the drama group</p>
+        </article>
+        <article class="card">
+          <i class="fas fa-music fa-2x"></i>
+          <h3>Time with the Choir</h3>
+          <p>The pax chior will serve us with some dancable tunes to shake off our worries.</p>
+        </article>
+        <article class="card">
+          <i class="fas fa-briefcase-medical fa-2x"></i>
+          <h3>Healing and Deliverance</h3>
+          <p>This section will be for the spiritually endowed men of God to heal 
+            & deliver participants from their bondage.
+          </p>
+        </article>
+        <article class="card">
+          <i class="fas fa-video fa-2x"></i>
+          <h3>Choreography</h3>
+          <p>As always, the Heralds of Honour will serve us with a new dance moves we are all 
+            going to like.
+          </p>
+        </article>
+      </div>
+      <a href="#" class="mobile-button">JOIN PRAISYDRAM 2022</a>
+      <a href="#" class="desktop-button">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <footer class="footer">
+      <div class="top">
+        <h2>Partners</h2>
+        <span class="border-span"></span>
+        <ul>
+          <li>Vodafone Ghana</li>
+          <li>Voltic Ghana</li>
+          <li>Coca-Cola Ghana</li>
+          <li>Pax Romana Ghana</li>
+        </ul>
+      </div>
+      <div class="bottom">
+        <div>
+          <img src="./public/assets/icons/pd-logo.png" alt="PraisyDram logo">
+        </div>
+        <p>2022 PraisyDram <br> Some Rights Reserved</p>
+      </div>
```

Comment:
- [ ] Your footer appearing differently from the design, Please make sure your footer content is in the center and the footer logo and text have the same margin from left and right

### Your design

![image](https://user-images.githubusercontent.com/83803414/157692002-f76bb545-2e2b-43c1-80e0-1151784d1bda.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/157692083-2a0eb7a3-bf3b-460d-9092-ab386e056736.png)